samples/deploy/rbac/apisix_view_clusterrole.yaml

```diff
@@ -137,6 +137,7 @@ rules:
   - apisixroutes
   - apisixupstreams
   - apisixservices
+  - apisixtlses
```

Comment:
ditto.

Reply:
It is the requirement of CRD , requires to be plural.

Reply:
Sigh, OK